hypervisor/arch/x86/vmx.c

```diff
@@ -958,9 +958,14 @@ static void init_entry_ctrl(__unused struct vcpu *vcpu)
 	 * on VM entry processor is in IA32e 64 bitmode * Start guest with host
 	 * IA32_PAT and IA32_EFER
 	 */
+#ifdef HV_DEBUG
+	value32 = (VMX_ENTRY_CTLS_LOAD_EFER |
+		   VMX_ENTRY_CTLS_LOAD_PAT |
+		   VMX_ENTRY_CTLS_LOAD_DBG);
+#else
```

Comment:
Do we need to set VMX_EXIT_CTLS_SAVE_DBG in vm exit control?
Do we need to save/restore DR0, DR1, DR2, DR3, DR6 in vm_entry and vm_exit?